app/models/metric/purging.rb

```diff
@@ -35,15 +35,14 @@ def self.purge_realtime_timer(ts = nil)
     purge_timer(ts, "realtime")
   end
 
+  # NOTE: purge_#{interval} was introduced to make uniqueness easier
+  # now that we do not require unique purging, possibly go back to :args => [value, interval]
```

Comment:
I feel like these NOTEs to yourself (as well as the others in this PR) would be better served if moved into a separate github issue.